libs/use-viewer/src/index.ts

```diff
@@ -182,7 +184,7 @@ const useViewer = ({ handleError, streamAccountId, streamName, subscriberToken }
     const { audio, video } = statistics.input;
 
     const mainAudio = audio.filter(({ mid }) => mid === mainAudioMIDRef.current) ?? [];
-    const mainVideo = video.filter(({ mid }) => mid === mainVideoMIDRef.current) ?? [];
+    const mainVideo = video.filter(({ trackIdentifier }) => trackIdentifier === mainVideoTrackIdRef.current);
```

Comment:
@vincentsong can probably comment more on the correctness of the logic, but I would encourage a comment on here explaining why we're using `trackIdentifier` instead of `mid`. As the repo is open source and used by the general public as reference guide when they build their own RTS based viewers, a description here would help them understand the rationale behind `trackIdentifier`, even if it's something as simple as `Safari and Firefox do not report on mid so we fall back to trackIdentifier to easily pick up stats when streaming with multiple sources`

Reply:
Sure thing, makes total sense.